source/usage-examples/command.txt

```diff
@@ -4,16 +4,52 @@ Run a Command
 
 .. default-domain:: mongodb
 
-You can run all raw database operations using the
-``MongoDatabase.runCommand()`` method. A raw database operation is a
-command you can execute directly on the MongoDB server CLI. These
-commands include administrative and diagnostic tasks, such as fetching
-server stats or initializing a replica set. Call the ``runCommand()``
-method with a Bson command object on an instance of a ``MongoDatabase``
-to run your raw database operation.
+You can run commands directly on the MongoDB server CLI using the
+``RunCommand()`` method. 
+
+The following example passes a document containing the ``"dbStats"``
+command to the ``RunCommand()`` method, which requests for statistics
+about the ``sample_restaurants`` database: 
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/command.go
+   :start-after: begin runCommand
+   :end-before: end runCommand
+   :emphasize-lines: 5
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/command.go>`__ to see a fully runnable example.
+
+Expected Result
+---------------
+
+After running the preceding code snippet, the ``result`` variable should
+contain the following: 
+
+.. code-block:: json
+
+    // results truncated
+    [ {db sample_restaurants} 
+      {collections 2} 
+      {views 0} 
+      {objects 25554} 
+      {avgObjSize 548.4101901854896} 
+      {dataSize 14014074} {storageSize 8257536} 
+      ... ]
 
 .. note::
-    Use the :manual:`mongo shell </reference/mongo-shell>` for
-    administrative tasks instead of the Go driver whenever possible,
-    since these tasks are often quicker and easier to implement with the
-    shell than in a Go application.
+
+    The ``result`` variable may vary depending on your data set.
```

Comment:
```suggestion
    The ``result`` variable may vary depending the contents of your collection.
```
Nit: This struck me as odd as the data set we are providing is in theory the same for everyone. I think this has more to do with the readers "collection" (specific of what is on their machine right now) than their "data set" (which in my head is the thing they downloaded from atlas the first time).